packages/tkg-clusterclass-aws/bundle/config/upstream/overlay-podsecuritystandard.yaml

```diff
@@ -0,0 +1,115 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@overlay/match by=overlay.subset({"kind":"ClusterClass"})
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: ClusterClass
```

Comment:
Naive question: is there some way to put this under [tkg-clusterclass](https://github.com/vmware-tanzu/tanzu-framework/tree/main/packages/tkg-clusterclass) to avoid needing to duplicate this info across each of the infrastructure providers?

Reply:
Great idea, wasn't aware that this is possible.

Moved to a separate secret via `tkg-clusterclass` which is used as overlay for the provider specific packageinstall.

Reply:
@zjs question: I now moved the overlay to the `tkg-clusterclass` package.

However: when creating a management cluster the current code seems to **not** use the package `tkg-clusterclass` package (at least it does not for `--dry-run` and at the current code state).

I want to confirm on the behaviour as soon as the CAP* Provider PRs are in to verify it.

Reply:
Correction: when integrating overlays into `tkg-clusterclass`:
* `tanzu mc create --dryrun` won't include the overlays
* `tanzu mc create --dryrun=false` will use the packageinstall and the overlays will apply.